src/components/TxDetailsModal.js

```diff
@@ -78,7 +78,6 @@ class TxDetailsModal extends Component {
       copyText: ncCallerAddr
     });
     const isNc = tx.isNanoContract();
-    const hasFirstBlock = tx.hasFirstBlock();
```

Comment:
I think we should add it back. If we do this, we should change below as well

`{isNc && isVoided` must remove the `isVoided`

Reply:
Done